README.md

```diff
@@ -1,3 +1,7 @@
+# Deprecation Notice
+**`pyhoma` is deprecated. Please use `pyoverkiz` 1.0 which is backwards compatible. See https://github.com/iMicknl/python-overkiz-api.**
```

Comment:
Correct me if I’m wrong, but Github will automatically redirect from python-tahoma-api to python-overkiz-api repository. So I don’t think the last part is required.

Reply:
I added this for pypi, so we link them to our GitHub from there as well.